### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: confd/production.tmpl [hunk 1]
```diff
@@ -23,9 +23,8 @@
     "probe": {
         "host_ip": "{{ getv "/probe/host/ip" "0.0.0.0" }}"
     },
-    "es": {
-        "index": "{{ getv "/es/index" "gpkgs_index" }}",
-        "host_ip": "{{ getv "/es/host/ip" "10.28.11.49" }}",
-        "port": "{{ getv "/es/port" "8080" }}"
+    "commonstorage": {
+        "host_ip": "{{ getv "/commonstorage/host/ip" "10.28.11.49" }}",
```
Comment (asafMasa) [inline confd/production.tmpl]: change to url (as in trigger)

File: confd/production.tmpl [hunk 2]
```diff
@@ -23,9 +23,8 @@
     "probe": {
         "host_ip": "{{ getv "/probe/host/ip" "0.0.0.0" }}"
```
Comment (asafMasa) [inline confd/production.tmpl]: i think the probe config can be removed

File: confd/production.tmpl [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
     "kafka" : {
         "host_ip": "{{ getv "/kafka/host/ip" "10.28.11.49:9092" }}",
-        "topic": "{{ getv "/kafka/topic" "topic-test-132" }}",
+        "topic": "{{ getv "/kafka/topic" "topic-test-888" }}",
         "group_id": "{{ getv "/kafka/group/id" "task_group_1" }}",
-        "auto_commit": "{{ getv "/kafka/auto/commit" "False" }}",
+        "auto_commit": {{ getv "/kafka/auto/commit" "false" }},
```
Comment (asafMasa) [inline confd/production.tmpl]: is there a reason for this param (auto_commit) to be config?

File: log/logger.py [hunk 4]
```diff
@@ -18,7 +18,7 @@ def _create_file_handler(self):
         if self.log.handlers:
             self.log.handlers = []
         file_handle = logging.FileHandler(self.__config['logger']['filename'])
-        file_handle.setStream(stdout)
+        #file_handle.setStream(stdout)
```
Comment (asafMasa) [inline log/logger.py]: ?

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: confd/production.tmpl (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply confd/production.tmpl]: i guess not, seems like its should be always False.

**Review by asafMasa — APPROVED** [review]

**Pull request merged**